surefire-its/src/test/resources/junit-platform/pom.xml

```diff
@@ -70,4 +64,17 @@
             </snapshots>
         </repository>
     </repositories>
+
+    <pluginRepositories>
```

Comment:
There are no snapshot plugins. Why you added this section?

Reply:
In `JUnitPlatformEnginesIT` there's the line 66 with `args.add( new Object[] { "1.4.0-SNAPSHOT", "5.4.0-SNAPSHOT", "1.1.1", "1.0.0" } );` being passed to `unpack( "junit-platform",...` within the `@Test platform()` method.
iirc, the auto-resolve feature didn't work when only a user-defined snapshot repo was added to the project.

Just double-checked and now it works. Removing the `<pluginRepositories>` element.